config/api.yml

```diff
@@ -35,6 +35,13 @@
     - :post
     - :patch
     - :delete
+    :gpppdo: &gpppdo
```

Comment:
@jntullo unfortunately we're only part way through the big refactor, so either that would have to come first or it would be `base#options` for now and I'd incorporate thta later. @abellotti any thoughts?